modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -429,6 +483,14 @@ public void defineNode(BLangNode node, SymbolEnv env) {
         this.env = prevEnv;
     }
 
+    public void defineSymbol(DiagnosticPos pos, BSymbol symbol, SymbolEnv env) {
```

Comment:
Can we use the existing defineSymbol method?

Reply:
Btw, you need to check for duplicates before defining a symbol

Reply:
Removed the method.